src/content/actions/test/unit/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "env": {
+      "mocha": true
+    }
+}
```

Comment:
mocha functions are automatically injected by the mocha runner, so this eslint configuration tells this to eslint. (we should likely add it to other test directories... or have only one test directory :) )